src/Microsoft.CodeAnalysis.BannedApiAnalyzers/Core/SymbolIsBannedAnalyzer.cs

```diff
@@ -59,8 +59,13 @@ public abstract class SymbolIsBannedAnalyzer<TSyntaxKind> : SymbolIsBannedAnalyz
                 where sourceText != null
                 from line in sourceText.Lines
                 let text = line.ToString()
-                where !string.IsNullOrWhiteSpace(text) && text[0] != '#'
-                select new BanFileEntry(text, line.Span, sourceText, additionalFile.Path);
+                where !string.IsNullOrWhiteSpace(text)
```

Comment:
Is this needed? We will already try to trim the comment and call `IsNullOrWhiteSpace` in a later null.

I think `text` will never be null here so only the whitespace check is relevant and is duplicated below.

Reply:
Good spotting, all the unit tests pass without the first `!string.IsNullOrWhitespace()` check. I'll remove that.

Reply:
Hi, is there anything else that is blocking this being merged?